keyboards/handwired/not_so_minidox/serial.c

```diff
@@ -0,0 +1,228 @@
+/*
+ * WARNING: be careful changing this code, it is very timing dependent
+ */
+
+#ifndef F_CPU
+#define F_CPU 16000000
+#endif
+
+#include <avr/io.h>
+#include <avr/interrupt.h>
+#include <util/delay.h>
+#include <stdbool.h>
+#include "serial.h"
+
+#ifdef USE_SERIAL
```

Comment:
Use `#ifndef USE_I2C` here. 

Otherwise, I will be changing it later, since this breaks config.qmk.fm compiling.

Reply:
should I add that line, or replace the 3 you've highlighted?

Reply:
You should replace the `#ifdef USE_SERIAL` line with `#ifndef USE_I2C`, specifically